mailbox/api/src/main/java/org/apache/james/mailbox/quota/QuotaManager.java

```diff
@@ -57,7 +57,7 @@ public Quota<QuotaSizeLimit, QuotaSizeUsage> getStorageQuota() {
      *
      * @param quotaRoot Quota root argument from RFC 2087 ( correspond to the user owning this mailbox )
      */
-    Quota<QuotaCountLimit, QuotaCountUsage> getMessageQuota(QuotaRoot quotaRoot) throws MailboxException;
+    Quota<QuotaCountLimit, QuotaCountUsage> getMessageQuota(QuotaRoot quotaRoot) throws OverQuotaException;
```

Comment:
You do not get a OverQuotaException by simply checking your quota, this do not make sense